lib/galaxy/webapps/galaxy/api/genomes.py

```diff
@@ -15,15 +18,14 @@ class GenomesController(BaseAPIController):
     RESTful controller for interactions with genome data.
     """
 
-    @web.legacy_expose_api_anonymous
+    @expose_api_anonymous
```

Comment:
🎆 

The other backend migration we're halfway through, any chance you can make an issue to track our remaining instances of legacy decorators? Backend team is rocking these checklists 😄.

Reply:
Of course! Writing it up now.

Reply:
@jmchilton, just to clarify, do you mean all instances of the `legacy*` decorators used in `galaxy/webapps`? (seem to be 88 instances of usage)?

Reply:
Yup - thanks!